packages/app_center/lib/widgets/app_card.dart

```diff
@@ -80,7 +80,7 @@ class AppCard extends StatelessWidget {
     final l10n = AppLocalizations.of(context);
     final cardLabel = [
       '${title.title}.',
-      title.publisher != null
+      title.publisher != null && title.publisher != ''
```

Comment:
Nitpick: `isNotEmpty` might make this slightly more readable
```suggestion
      title.publisher?.isNotEmpty ?? false
```

Reply:
Always forget about those utility fields, thanks :)